streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -625,6 +635,15 @@ private void runLoop() {
         log.info("{} Shutting down at user request", logPrefix);
     }
 
+    private void reset(final TopicPartition partition, final Set<TopicPartition> partitions, final String logMessage, final String resetPolicy, final Set<String> loggedTopics) {
```

Comment:
nit: maybe change method name to `addToResetList` since resetting does not occur here?